### PR TITLE
Bug/#35816 mi2 mysql keyword error

### PR DESCRIPTION
**Pull request by fsacha**

### CODE REVIEW

File: db/upgrade.php [hunk 1]
```diff
@@ -91,5 +91,15 @@ function xmldb_auth_mumie_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2023062000, 'auth', 'mumie');
     }
 
+    if ($oldversion < 2023110800) {
+        $table = new xmldb_table('auth_mumie_cryptographic_key');
+        $field = new xmldb_field('key');
+        if ($dbman->field_exists($table, $field)) {
+            $field->set_attributes(XMLDB_TYPE_TEXT, null, null, XMLDB_NOTNULL, null, null);
```
Comment (dung238) [inline db/upgrade.php:98]: what is the reason for setting these attributes?

Comment (fsacha) [reply db/upgrade.php:98]: Otherwise its not working, you will get an exception if you skip it.

Comment (dung238) [reply db/upgrade.php:98]: so did the attributes change?

Comment (fsacha) [reply db/upgrade.php:98]: No, just the field name :)

**Pull request merged**